src/System.Data.SqlClient/src/System/Data/SqlClient/SqlConnection.cs

```diff
@@ -48,6 +48,8 @@ public sealed partial class SqlConnection : DbConnection, ICloneable
         // using SqlConnection.Open() method. 
         internal bool _applyTransientFaultHandling = false;
 
+        internal SqlTransaction CurrentTransaction { get; set; }
```

Comment:
How does this "fix" the problem for System.Transactions.Transaction? Is the System.Transactions.Transaction wrapped inside of the SqlTransaction?